### PR TITLE
chore: update node to version 20

### PR DESCRIPTION
**Pull request by kilted-andres**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ You have three different options:
 1. Using `yarn environment`:
 
    There is a script to facilitate the generation of the environment variables inside of this repository.
-   This script is called `./scripts/genesisEnvironmentVariables.ts`.
+   This script is called `./launchScripts/genesisEnvironmentVariables.ts`.
```
Comment (Dudleyneedham) [inline README.md:164]: Why is this part of the changes?

File: launchScripts/genesisWellKnownDidConfig.ts [hunk 2]
```diff
@@ -204,7 +204,7 @@ async function main() {
   await Kilt.disconnect()
 
   console.log(
-    'this is the well-known-DID-configuration of your dApp that was just created: \n',
+    "this is your dApp's Domain Linkage Credential (well-known-DID-configuration) that was just created: \n",
```
Comment (Dudleyneedham) [inline launchScripts/genesisWellKnownDidConfig.ts:207]: This shouldn't be part of the changes, we can make a separate ticket for this

File: package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     "backend"
   ],
   "engines": {
-    "node": ">= v18.17.0"
+    "node": ">= v20.13.1"
```
Comment (Dudleyneedham) [inline package.json:15]: Was there any updates on the `yarn.lock`

**Review by Dudleyneedham — CHANGES_REQUESTED** [review]
Lets keep this to updating the node version, but it seems you didn't add `yarn install`

File: package.json (hunk 3 above)
Comment (kilted-andres) [reply package.json:15]: It seems like all packages where compatible with both `node` versions, so `yarn` did not had to update any packages.

File: launchScripts/genesisWellKnownDidConfig.ts (hunk 2 above)
Comment (kilted-andres) [reply launchScripts/genesisWellKnownDidConfig.ts]: Move it to #84

File: README.md (hunk 1 above)
Comment (kilted-andres) [reply README.md]: Moved it to #84

**Pull request merged**